api/src/main/resources/workbench.yaml

```diff
@@ -1925,3 +1936,20 @@ definitions:
         format: int32
         description: >
           Number of clusters deleted during the check.
+
+  RecentResourceResponse:
+    type: array
+    items:
+      $ref: "#/definitions/RecentResource"
+
+  RecentResource:
+    type: object
+    properties:
+      permission:
+        type: string
+      cohort:
+        $ref: "#/definitions/Cohort"
+      notebook:
+        $ref: '#/definitions/FileDetail'
+      modifiedTime:
```

Comment:
lastAccessedTime, this is different from last modified which we are tracking separately.

I also think we should consider making this a field on the notebooks / cohorts.

Reply:
Do you mean inside notebook or cohort object? lastAccesstime or modified time is user based, that can be different from the actual notebook or cohort last modified time

Reply:
Good point - this is a good argument for keeping it outside. My concern was that we might need the same data for the existing notebooks/cohorts pages (and on the workspace about page). But perhaps you hvae another plan for getting that into there.